pkg/operator/ceph/cluster/cluster.go

```diff
@@ -604,14 +604,18 @@ func (c *cluster) configureMsgr2() error {
 	monStore := config.GetMonStore(c.context, c.ClusterInfo)
 	if c.Spec.Network.Connections.Encryption != nil {
 		encryptionSetting := "crc secure"
+		// crc mode, if denied agree to secure mode
+		defaultRBDMapOptions := "ms_mode=prefer-crc"
```

Comment:
Should we just default to `crc`? Or why would crc be denied to fall back to secure? 
```suggestion
defaultRBDMapOptions := "ms_mode=crc"
```

Reply:
@travisn i chose this one because of the encryption setting `encryptionSetting := "crc secure"`. Please let me know if you want me to modify this to crc?

Reply:
I see, it does match that behavior of the other encryption setting. Sounds good to keep it, I would expect it will always use crc in that case anyway, and wouldn't fall back to secure.